### ISSUE TITLE
Tutorial data is not accessible when running from github repo with docker

### ISSUE THREAD

**Issue by mheuer-nmdp** [nmdp-bioinformatics/flow#18]
See
http://www.nextflow.io/docs/latest/sharing.html#bootstrap-data

**Comment by mheuer-nmdp** [comment]
Fixed by #19

**Comment by mheuer-nmdp** [comment]
Reopened; while the pipeline completes successfully, the VCF files are empty.  bwa can't access the reference data parameter
```bash
$ cat .command.err 
[E::bwa_idx_load_from_disk] fail to locate the index files
[fai_load] build FASTA index.
[fai_build] fail to open the FASTA file /home/nextflow/.nextflow/assets/nmdp-bioinformatics/flow/tutorial/ref/chr6-ex.fa
[fai_load] fail to open FASTA index.
```
